crates/fluvio-version-manager/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "fluvio-version-manager"
+version = "0.1.0"
+edition = "2021"
+authors = ["Fluvio Contributors <team@fluvio.io>"]
+description = "Fluvio Version Manager"
+repository = "https://github.com/infinyon/fluvio"
+license = "Apache-2.0"
+publish = false
+
+[[bin]]
+name = "fvm"
+path = "src/main.rs"
+
+[dependencies]
+dialoguer = "0.11.0"
+
+# Workspace Dependencies
+async-std = { workspace = true, features = ["attributes"] }
+clap = { workspace = true , features = ["std", "color", "derive", "help"] }
+colored = { workspace = true }
+color-eyre = { workspace = true, default-features = false }
+comfy-table = { workspace = true }
+dirs = { workspace = true }
+hex = { workspace = true }
+semver = { workspace = true }
+serde = { workspace = true }
+serde_json = { workspace = true }
+sha2 = { workspace = true }
+surf = { workspace = true }
```

Comment:
this should not have access to `surf` since all downloading thru should delegated to `hub-util`

Reply:
Gotcha! Yes Im not using it here, its from migration C/P. Removing immediately!